cmd/hyperconverged-cluster-operator/main.go

```diff
@@ -168,6 +173,14 @@ func main() {
 	err = createPriorityClass(ctx, mgr)
 	cmdHelper.ExitOnError(err, "Failed creating PriorityClass")
 
+	// Setup Monitoring
+	operatormetrics.Register = controllerruntimemetrics.Registry.Register
+	err = metrics.SetupMetrics()
+	if err != nil {
+		log.Printf("failed to setup metrics: %v", err)
```

Comment:
I think you should use `logger.Error` here as it is being used across the file in case of an error.

Reply:
I followed syntax we did in other operators, @machadovilaca WDYT?

Reply:
OK, but if you look in this file, `log` package never existed. Your PR introduces an import for it. It's not wrong or bad. It's just that we're doing the same thing, that you're doing here, using `logger`. So, IMO, you could just follow the same. I'll let Joao chime in. :)

Reply:
+1 thanks for explaining, so I agree with changing that if @machadovilaca agrees as well (:

Reply:
`I followed syntax we did in other operators` -> we should only try to keep the similarity between the `pkg/monitoring` packages of the different operators, we should still use the packages/conventions they use in these cases